packages/shared/src/components/marketingCta/MarketingCtaList.tsx

```diff
@@ -1,16 +1,10 @@
 import React, { ReactElement, useCallback, useEffect, useRef } from 'react';
-import { ListCard } from './ListCard';
-import { useBoot } from '../../../hooks';
-import { useLogContext } from '../../../contexts/LogContext';
-import { LogEvent, TargetType } from '../../../lib/log';
-import {
-  CTAButton,
-  Description,
-  Header,
-  MarketingCta,
-  Title,
-} from '../MarketingCta/common';
-import { CardCoverList } from './CardCover';
+import { ListCard } from '../cards/list/ListCard';
```

Comment:
Yeah 🙈
I like them being grouped together, in both ways. Though I would probably lean towards the latter method, especially since CTAs can have so many different forms which would make all components spread out.

Reply:
Yeah makes sense, I'm ok with the changes here